New package: CopernicusData v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: CopernicusData
- Repository: https://github.com/vlevasseur073/CopernicusData.jl
- Created by: @vlevasseur073
- Version: v0.1.0
- Commit: e26b221f5ed3a186f9edc95a64df5349aadcac2c
- Reviewed by: @vlevasseur073
- Reference: https://github.com/vlevasseur073/CopernicusData.jl/issues/13#issuecomment-2897643175
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359843431edc2e9df6e55a6dff2b4f6102a1db7abdc925a16b6d5ab2aa5a3f9b8fde6e3dd97050a4816279cc496497ab97e4436551dda103e1352630336cffcbcebd64053188295706ee0a49007557d4a0338b2a3fd518d9df2b636f0efa7fd46274fe3088578da9eef72bd675d7548af6a1444b1103b08724d869cecbb21037a0ff -->

goerz commented:
[noblock] When fixing automerge issues, please keep the version number the same. That way, it updates the existing PR

JuliaRegistrator committed:
New package: CopernicusData v0.1.0

UUID: 40f541ec-ef23-43f7-b2aa-6425eeeec50a
Repo: https://github.com/vlevasseur073/CopernicusData.jl.git
Tree: 7c70b6163c94f920d61afd987d8d7c6b3b6ad75f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
